File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/H. Two numbers/main.dart
import 'dart:io';

void main() {
  List<String> input = stdin.readLineSync()!.split(" ");
  int a = int.parse(input[0]);
  int b = int.parse(input[1]);
  double div = a / b;

  print("floor ${a} / ${b} = ${div.floor()}");
  print("ceil ${a} / ${b} = ${div.ceil()}");
  print("round ${a} / ${b} = ${div.round()}");
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/H. Two numbers/main.cs
string[] input = Console.ReadLine().Split(" ");
int a = Convert.ToInt32(input[0]);
int b = Convert.ToInt32(input[1]);
float div = (float)a/b;

Console.WriteLine($"floor {a} / {b} = {Math.Floor(div)}");
Console.WriteLine($"ceil {a} / {b} = {Math.Ceiling(div)}");
Console.WriteLine($"round {a} / {b} = {Math.Round(div, MidpointRounding.AwayFromZero)}");